image/templates/helm/stackrox-central/values-private.yaml.example

```diff
@@ -176,3 +176,51 @@
 #       -----BEGIN RSA PRIVATE KEY-----
 #       <PEM-encoded Scanner DB service cert private key>
 #       -----END RSA PRIVATE KEY-----
+#
+# # Secret configuration options for the Scanner V4 deployment.
+# scannerV4:
+#   indexer:
+#     # Internal "scanner-v4-indexer.<namespace>.svc" service TLS certificate for the Scanner V4 indexer deployment.
+#     # Omit to auto-generate (initial deployment) or use existing (upgrade).
+#     serviceTLS:
+#       cert: |
+#         -----BEGIN CERTIFICATE-----
+#         <PEM-encoded Scanner V4 indexer service certificate>
+#         -----END CERTIFICATE-----
+#       key: |
+#         -----BEGIN RSA PRIVATE KEY-----
+#         <PEM-encoded Scanner V4 indexer service cert private key>
+#         -----END RSA PRIVATE KEY-----
+#
+#   matcher:
+#     # Internal "scanner-v4-matcher.<namespace>.svc" service TLS certificate for the Scanner V4 matcher deployment.
+#     # Omit to auto-generate (initial deployment) or use existing (upgrade).
+#     serviceTLS:
+#       cert: |
+#         -----BEGIN CERTIFICATE-----
+#         <PEM-encoded Scanner V4 matcher service certificate>
+#         -----END CERTIFICATE-----
+#       key: |
+#         -----BEGIN RSA PRIVATE KEY-----
+#         <PEM-encoded Scanner V4 matcher service cert private key>
+#         -----END RSA PRIVATE KEY-----
+#
+#   db:
+#     # The password to be used for authenticating database access. This is not user-relevant
+#     # and only serves to properly secure the database with a pre-shared secret. If this
```

Comment:
Same comment with the `user-relevant` I mentioned for secured cluster chart.

Reply:
✅